shared/templates/rsyslog_logfiles_attributes_modify/bash.template

```diff
@@ -7,7 +7,14 @@ RSYSLOG_ETC_CONFIG="/etc/rsyslog.conf"
 #   (store the result into array for the case there's shell glob used as value of IncludeConfig)
 readarray -t OLD_INC < <(grep -e "\$IncludeConfig[[:space:]]\+[^[:space:];]\+" /etc/rsyslog.conf | cut -d ' ' -f 2)
 readarray -t RSYSLOG_INCLUDE_CONFIG < <(for INCPATH in "${OLD_INC[@]}"; do eval printf '%s\\n' "${INCPATH}"; done)
+
```

Comment:
@ComplianceAsCode/red-hatters @ComplianceAsCode/suse-maintainers @ComplianceAsCode/oracle-maintainers 
according to our documentation: https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#bash
`Prefer to use sed rather than awk.`

Should we instead make the provided solution here the default rather than having this if else? Thoughts?

Reply:
I agree, a unified solution that works across all products is better.

Reply:
As suggested, I switched to using a unified solution for all products.
I also made minor changes to the regex and added case insensitivity to grep searches.